ds_judgements_public_ui/templates/includes/judgment_text_toolbar.html

```diff
@@ -23,15 +23,9 @@ <h1 class="judgment-toolbar__title">{{ document|get_title_to_display_in_html }}<
           </a>
         {% endspaceless %}
       {% endif %}
-      <a class="judgment-toolbar-buttons__option--pdf btn"
```

Comment:
We need to make sure this unused CSS class is removed from the SCSS as well.

Reply:
removed updated now